src/index.js

```diff
@@ -132,9 +44,4 @@ clsBtn.addEventListener('click', (event) => {
   del();
 });
 
-inp.addEventListener('keyup', (event) => {
-  if (event.key === 'Enter') {
-    inp.value = '';
-    display();
-  }
-});
\ No newline at end of file
+display();
```

Comment:
- [optional] ( ***but mandatory*** ). Good job implementing what the previous code reviewer requested by allowing your change state to persist in using local storage and updating it accordingly, However, I notice that for changing a task to complete, it saves on the local storage as intended, but on reloading the strike-through on the completed task goes off ( Which suppose to indicate that the is completed), but the local storage still holds the correct true value for that task.

**In Summary** 
- please make sure your interface uses the data from the local storage for showing completed/incomplete tasks, on reload of the page. 


#### The video below shows more insight
- You can see how the task changes to true on the local storage when I mark them as complete and there are stricken through, but when I reload the struckthrough leaves but the tasks are still `true` as they should be on your local storage.
- 
[Screencast from 05-13-2023 06:54:41 PM.webm](https://github.com/faisalakb/Todo_List/assets/58771507/cb7fb5fc-2091-49b3-b5da-7a5d2db22b39)